Tests/IntegrationTests.Shared/StandAloneObjectTests.cs

```diff
@@ -85,5 +82,90 @@ public void RealmObject_WhenStandalone_ShouldHaveDefaultEqualsImplementation()
 
             Assert.DoesNotThrow(() => _person.Equals(otherPerson));
         }
+
+        [Test]
+        public void RealmObject_WhenManaged_ShouldNotThrow()
```

Comment:
Sure, I will take a look at the WeaverTests infrastructures and try to come up with one.